### DISCUSSION TITLE
Is is safe to use the same data directory from both the desktop and server versions of Trilium?

### DISCUSSION THREAD

**Discussion by myii** [TriliumNext/Trilium#1409] [Q&A] [answered]
Following on from #1407, with respect to the [`trilium-data` directory](https://github.com/zadam/trilium/wiki/Data-directory), can the same directory be used for both the desktop and server editions of Trilium (obviously when running from the same machine)?

So Trilium Deskop already uses `~/.local/share/trilium-data`.  Is it safe to use `-v ~/.local/share/trilium-data:/root/trilium-data` when using the Docker-based server?  Or should [Synchronization](https://github.com/zadam/trilium/wiki/Synchronization) be used in all cases?

**Comment by zadam** [comment, marked as answer]
Only if they are not running at the same time - one document can't be accessed by more than 1 process.

If you want to have 2 trilium processes running, then you need to setup synchronization.

**Reply by myii** [reply]
Excellent, thanks for the prompt response!
